src/view/actions/applyPropositions.jsx

```diff
@@ -50,20 +88,59 @@ const applyPropositionsForm = form({}, [
         description: "Enter your scope",
         validationSchemaBase: string().required("Please provide a scope.")
       }),
-      textField({
-        name: "selector",
-        label: "Selector",
-        description: "Enter your selector",
-        validationSchemaBase: string().required("Please provide a selector.")
+      radioGroup({
+        name: "elementType",
+        label: "Target element",
+        items: [
+          { value: "selector", label: "Selector" },
+          { value: "element", label: "Element" }
```

Comment:
The `trackProposition` command did support an element or selector to be passed in, but `applyPropositions` does not support this.  It relies on selectors only.  So element should be removed here.